proteus/Archiver.py

```diff
@@ -1345,7 +1357,7 @@ def writeMeshXdmf_C0Q2Lagrange(self,ar,name,mesh,spaceDim,dofMap,t=0.0,init=Fals
                         if init or meshChanged:
                             if ar.has_h5py:
                                 ar.create_dataset_sync('elements'+spaceSuffix+`tCount`,
-                                                       offsets = mesh.globalMesh.elementOffsets_subdomain_owned*nsubelements,
+                                                       offsets = [x*nsubelements for x in mesh.globalMesh.elementOffsets_subdomain_owned],
```

Comment:
BTW,  you can write this more concisely as  `mesh.globalMesh.elementOffsets_subdomain_owned*nsubelements`